scripts/exclude-links.txt

```diff
@@ -0,0 +1,20 @@
+http://127.0.0.1:10000/devstoreaccount1;
+http://localhost:3000/
+https://$
+https://api.github.com/repos/$
+https://blog.$
+https://discuss.$
+https://dvc.org/some.link
+https://example.com/data.txt
+https://example.com/path/to/data
+https://example.com/path/to/data.csv
+https://example.com/path/to/dir
+https://github.com/$
```

Comment:
why do we exclude this one?

Reply:
there's code definitions - in `js` files things like

```js
const src = `https://github.com/${repo}`;
```

but I can't think of a way to filter them all out in a way that will not also filter out certain cases from markdown files...

Reply:
oh, I see it's literally `$` ... not end of line